src/popup/components/Main.vue

```diff
@@ -209,6 +234,10 @@ export default {
       searchString: "",
       aliasArray: [], // array of existing alias
       hasLoadMoreAlias: true,
+      moreOptions: {
+        index: -1,
+        loading: false,
```

Comment:
should we rename `loading` to `disableDeleteButton` to be more explicit about its purpose?